app/controllers/internal/events_controller.rb

```diff
@@ -1,8 +1,40 @@
 module Internal
   class EventsController < ::InternalController
+    layout "internal"
+
     def index
-      # stubbed for now
-      head :ok
+      @no_results = load_pending_events.blank?
```

Comment:
I would move the `load_pending_events` to a `before_action` and have:

```
@no_results = @events.blank?
```

Otherwise someone may inadvertently remove the no results variable and not realise that the events will stop displaying